.github/workflows/release.yaml

```diff
@@ -89,3 +89,37 @@ jobs:
           file: ${{ steps.attest.outputs.bundle-path}}
           tag: ${{ github.ref }}
           overwrite: true
+
+  docs:
+    runs-on: ubuntu-latest
+    name: Build and publish documentation
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # pin@v4.2.2
+        with:
+          persist-credentials: false
+      - name: Environment Setup
+        uses: ./.github/actions/setup
+        with:
+          install: true
+          npm: true
+      - name: Install dependencies
+        run: |
+          pip install --require-hashes -r contrib/dev_reqs/requirements.txt
+          pip install --require-hashes -r docs/requirements.txt
```

Comment:
I think that you will also need to install `src/backend/requirements.txt` also (as per the `readthedocs.yml` file)

Reply:
Not necessary as we already need to run inv update